### PR TITLE
Changelog cli

### PR DESCRIPTION
**Pull request by rickducott**
Tested manually

### CODE REVIEW

**Comment by rickducott** [comment]
All the testing was manual, but seems to work. Want to test on testrepo tomorrow

File: docsutils/cli.go [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+	cmd := exec.Command("git", args...)
+	cmd.Dir = dir
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return string(output), nil
+}
+
+func replaceDirectories(product string, paths ...string) error {
```
Comment (EItanya) [inline docsutils/cli.go]: Accept fs arg here rather than creating new one

**Comment by EItanya** [comment]
> All the testing was manual, but seems to work. Want to test on testrepo tomorrow

**Comment by EItanya** [comment]
> All the testing was manual, but seems to work. Want to test on testrepo tomorrow

testrepo should be in a good state for testing tmrw morning

**Pull request merged**